### PR TITLE
Fix isort processing and rearrange setup.py imports

### PR DESCRIPTION
**Pull request by CartoonFan**
I disabled some unsupported isort settings so that isort will correctly process files again. Afterwards, I tested it out on setup.py, and things seem good so far.

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -10,14 +10,12 @@ skip=
...
     LOCALFOLDER
-min-similarity-lines=5
+;min-similarity-lines=5
 default_section=THIRDPARTY
-ignore-comments=yes
-ignore-docstrings=yes
-ignore-imports=yes
+;ignore-comments=yes
+;ignore-docstrings=yes
+;ignore-imports=yes
```
Comment (AlexanderRavenheart) [inline .isort.cfg:21]: Remove these lines entirely. These disabled options are no longer supported on isort 5.x.

Comment (CartoonFan) [reply .isort.cfg:21]: What about min-similarity-lines? Should I delete that too?

Comment (AlexanderRavenheart) [reply .isort.cfg:21]: Yes, support for that was also removed.

Comment (CartoonFan) [reply .isort.cfg]: Right, it's done.

**Review by AlexanderRavenheart — APPROVED** [review]

**Pull request merged**